src/main/scala/wdl4s/cwl/CwlEncoders.scala

```diff
@@ -0,0 +1,22 @@
+package wdl4s.cwl
+
+import eu.timepit.refined.api.Refined
+import io.circe.{Encoder, Json}
+import wdl4s.cwl.CwlType.CwlType
+import wdl4s.cwl.CwlVersion.CwlVersion
+import io.circe.syntax.EncoderOps
+
+/**
+  * wdl4s
+  * Created by oruebenacker on 06.07.17.
+  */
+object CwlEncoders {
```

Comment:
Looks like the Decoders moved to cwl.Implicits, so I suppose Encoders should go there, too?